pkg/551cb44b-5b4c-43c4-84ee-9044327b50c8/package.lp

```diff
@@ -0,0 +1,136 @@
+(librepcb_package 551cb44b-5b4c-43c4-84ee-9044327b50c8
+ (name "TCA9548A I2C-MUX")
```

Comment:
I'm not sure if the package is that specific to the TCA9548A breakout. Do the Adafruit breakout boards have similar pitch / pin counts?
For now I think we can just use "TCA9548A Breakout".

Reply:
I think adafruit boards all have the same pitch (2.54mm) but differ in number of pins...